node_modules/oae-core/comments/comments.html

```diff
@@ -3,7 +3,7 @@
 
 <!-- CONTENT -->
 <div class="comments-widget">
-    <ul id="comments-container" class="media-list"><!-- --></ul>
+    <ul id="comments-container" class="media-list" role="list"><!-- --></ul>
```

Comment:
I thought about this for awhile and concluded that it's best not to add the aria/role markup to the new comment grouping. The point of the aria/role markup is to expose the hierarchy of the comments, and a new comment form (which isn't even a comment yet) doesn't really fit in that hierarchy. I then checked the W3C specs to make sure that there was no _requirement_ to mark all `<li>` as `role="listitem"` and there isn't. The flow in screenreaders seems okay as well. First it announces a list, then it reads the new comment box as a list item but without a level. Then it reads the existing comments with their associated levels.